lte/gateway/c/oai/tasks/mme_app/mme_config.c

```diff
@@ -795,7 +795,27 @@ int mme_config_parse_file(mme_config_t *config_pP)
     config_pP->gummei.nb = 0;
     if (setting != NULL) {
       num = config_setting_length(setting);
-      OAILOG_INFO(LOG_MME_APP, "Number of GUMMEIs supported =%d\n", num);
+      OAILOG_INFO(LOG_MME_APP, "Number of GUMMEIs configured =%d\n", num);
+      if (!num) {
+        OAILOG_CRITICAL(
+          LOG_MME_APP,
+          "Not even one GUMMEI is configured, "
+          "configure minimum one GUMMEI \n");
+        DevMessage(
```

Comment:
Replace DevMessage with AssertFatal. The OAILOG_CRITICAL will be part of AssertFatal in PR#1135